charts/lfx-v2-project-service/templates/ruleset.yaml

```diff
@@ -22,23 +22,50 @@ spec:
           config:
             values:
               aud: lfx-v2-project-service
-    - id: "rule:lfx:lfx-v2-project-service:projects"
+    - id: "rule:lfx:lfx-v2-project-service:create_projects"
```

Comment:
Could we namespace these for better readability like:
`"rule:lfx:<service>:<component>:<resource>:<action>"`?

This would change the rules to the following:
```
rule:lfx:lfx-v2-project-service:projects:create
rule:lfx:lfx-v2-project-service:projects:list # to be removed
rule:lfx:lfx-v2-project-service:projects:get
rule:lfx:lfx-v2-project-service:projects:update
rule:lfx:lfx-v2-project-service:projects:delete
rule:lfx:lfx-v2-project-service:project_setings:get
rule:lfx:lfx-v2-project-service:project_setings:update
```